### DISCUSSION TITLE
How to prevent a specific package from auto upgrading ?

### DISCUSSION THREAD

**Discussion by NikhilNanjappa** [renovatebot/renovate#17937] [Request Help]
I have an organisation level framework package on a particular version as shown below. Renovate bot seems to be auto upgrading this package which is not intended (even minor version bump breaks our app).

So, how do I prevent renovate bot from not touching **only this specific package** ?

**package.json**
```
...
"@org/package-name": "7.0.7",
...
```

**renovate.json**
```
{
  "assigneesFromCodeOwners": "false",
  "automerge": true,
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "assigneesFromCodeOwners": "true",
      "automerge": false
    },
    {
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"]
    }
  ]
}
```

**Comment by NikhilNanjappa** [comment]
Ok, I just found `ignoreDeps` in the docs. Is that what I need in renovate.json ?

```
{
  ...
  "ignoreDeps": ["@org/package-name"],
  ...
}
```

It does seem like but any confirmation would be appreciated

**Reply by rarkins** [reply]
yes, that will ignore it completely

BTW your second package rule above is wrong though - it's just matching but not _doing anything_ once it's matched.

I also recommend that you don't set `automerge: true` at the top level. Instead write a package rule to opt into it for patch, minor, pin, digest.
